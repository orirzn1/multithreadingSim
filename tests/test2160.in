L10
S6
O8
N5

T0
I@0x00000000
ADDI $5, $1, 0x4
SUBI $5, $5, 0x3
ADDI $3, $1, 0x2
STORE $1, $1, $2
SUBI $5, $7, 0x1
STORE $0, $6, $2
SUBI $6, $7, 0x4
STORE $0, $1, 0x0
HALT $0

T1
I@0x00000000
SUB $4, $4, $6
ADDI $3, $1, 0x1
LOAD $4, $6, 0x2
LOAD $5, $6, 0x4
STORE $6, $3, 0x1
HALT $0

T2
I@0x00000000
LOAD $3, $2, $3
SUBI $7, $5, 0x2
ADD $7, $1, $4
SUB $3, $7, $7
SUBI $3, $6, 0x2
STORE $2, $0, 0x3
ADD $4, $1, $0
ADD $7, $2, $4
STORE $3, $2, $4
HALT $0

T3
I@0x00000000
ADDI $6, $3, 0x4
STORE $0, $5, $6
ADDI $0, $4, 0x1
SUB $0, $1, $2
SUB $2, $4, $7
SUBI $4, $7, 0x4
SUB $3, $4, $2
SUBI $7, $2, 0x2
HALT $0

T4
I@0x00000000
SUBI $3, $7, 0x2
STORE $6, $4, 0x1
LOAD $6, $2, $5
STORE $7, $7, $4
SUBI $1, $6, 0x2
SUBI $7, $0, 0x4
SUB $0, $0, $2
ADDI $7, $3, 0x3
SUB $0, $5, $5
LOAD $5, $3, 0x0
HALT $0

D@0x0
0x2
0x5
0x1
0x3
0x2
0x3
0x9
0xA
0x2
0x3
