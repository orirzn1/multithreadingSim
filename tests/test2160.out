
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x2	R4 = 0x0	R5 = 0xFFFFFFFF	R6 = 0xFFFFFFFC	R7 = 0x0
Register file thread id 1:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x1	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0xFFFFFFFE	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0xFFFFFFFC	R4 = 0xFFFFFFFC	R5 = 0x0	R6 = 0x4	R7 = 0xFFFFFFFE
Register file thread id 4:
	R0 = 0x0	R1 = 0xFFFFFFFE	R2 = 0x0	R3 = 0xFFFFFFFE	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x1
Blocked MT CPI for this program 4.244444

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x2	R4 = 0x0	R5 = 0xFFFFFFFF	R6 = 0xFFFFFFFC	R7 = 0x0
Register file thread id 1:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x1	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0xFFFFFFFE	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0xFFFFFFFC	R4 = 0xFFFFFFFC	R5 = 0x0	R6 = 0x4	R7 = 0xFFFFFFFE
Register file thread id 4:
	R0 = 0x0	R1 = 0xFFFFFFFE	R2 = 0x0	R3 = 0xFFFFFFFE	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x1
Finegrained Multithreading CPI for this program 1.244444

